src/libraries/System.Private.Xml/src/System/Xml/Xsl/Xslt/XsltLoader.cs

```diff
@@ -1944,18 +1943,7 @@ private VarPar XslVarPar()
             string select = ParseStringAttribute(1, "select");
             string asType = ParseStringAttribute(2, "as");
             TriState required = ParseYesNoAttribute(3, "required");
-            if (nodeType == XslNodeType.Param && _curFunction != null)
-            {
-                if (!_input.ForwardCompatibility)
-                {
-                    CheckError(required != TriState.Unknown, /*[???]*/SR.Xslt_RequiredOnFunction, name.ToString());
-                }
-                required = TriState.True;
-            }
-            else
-            {
-                if (required == TriState.True) ReportNYI("xsl:param/@required == true()");
-            }
+            if (required == TriState.True) ReportNYI("xsl:param/@required == true()");
```

Comment:
Nit, are we OK with single line if's or just following existing style?

Reply:
More than following the existing style, it's entirely the existing code... I just deleted the dead code around it.